### PR TITLE
MGMT-8423: Rename api-vip-connected validation

### PR DESCRIPTION
**Pull request by rollandf**
# Assisted Pull Request

## Description

Rename Day 2 host `api-vip-connected` validation to `ignition-downloadable`

## List all the issues related to this PR

- [ ] New Feature
- [x] Bug fix
- [ ] Tests
- [ ] Documentation
- [ ] CI/CD 

## What environments does this code impact?

- [ ] Automation (CI, tools, etc)
- [x] Cloud
- [x] Operator Managed Deployments
- [ ] None

## How was this code tested?

- [ ] assisted-test-infra environment
- [ ] dev-scripts environment
- [ ] Reviewer's test appreciated
- [x] Waiting for CI to do a full test run
- [ ] Manual (Elaborate on how it was tested)
- [ ] No tests needed

## Assignees

/cc @avishayt 
/cc @eranco74
/cc @nirfarkas 

## Checklist

- [x] Title and description added to both, commit and PR.
- [x] Relevant issues have been associated (see [CONTRIBUTING] guide)
- [x] Reviewers have been listed
- [x] This change does not require a documentation update (docstring, `docs`, README, etc)
- [x] Does this change include unit-tests (note that code changes require unit-tests)

## Reviewers Checklist

- [ ] Are the title and description (in both PR and commit) meaningful and clear?
- [ ] Is there a bug required (and linked) for this change?
- [ ] Should this PR be backported?

[Kubernetes community documentation]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines
[CONTRIBUTING]: https://github.com/openshift/assisted-service/blob/master/CONTRIBUTING.md

### CODE REVIEW

**Comment by rollandf** [comment]
@jkilzi @jelkosz Any impact on UI?

**Review by avishayt — APPROVED** [review]

File: swagger.yaml [hunk 1]
```diff
@@ -8530,7 +8530,7 @@ definitions:
       - 'hostname-unique'
       - 'hostname-valid'
       - 'belongs-to-machine-cidr'
-      - 'api-vip-connected'
+      - 'ignition-downloadable'
```
Comment (jkilzi) [inline swagger.yaml:8533]: @rollandf lgtm, we'll need just a fancy name for this validation...

Comment (rollandf) [reply swagger.yaml:8533]: > @rollandf lgtm, we'll need just a fancy name for this validation...

Do we need a Jira to follow up?

Comment (jkilzi) [reply swagger.yaml:8533]: Not sure. It is only being used [here](https://github.com/openshift-assisted/assisted-ui-lib/blob/d660d51055dfc018edbd57f0a3298f9ea9cc037f/src/common/config/constants.ts#L177)

**Pull request merged**